packages/web-api/src/response/ViewsPublishResponse.ts

```diff
@@ -2,12 +2,141 @@
 import { WebAPICallResult } from '../WebClient';
 export type ViewsPublishResponse = WebAPICallResult & {
   ok?:                boolean;
+  warning?:           string;
   error?:             string;
-  response_metadata?: ResponseMetadata;
   needed?:            string;
   provided?:          string;
+  view?:              View;
+  response_metadata?: ResponseMetadata;
 };
 
 export interface ResponseMetadata {
   messages?: string[];
 }
+
+export interface View {
+  id?:                    string;
+  team_id?:               string;
+  type?:                  string;
+  title?:                 Close;
+  submit?:                Close;
```

Comment:
I understand type wise that `title`, `submit`, and `close` are the same (I think so at least), but semantically, it seems really weird to have the types of `title` and `submit` be `Close`

Reply:
This is automatically determined by the generator tool. Unfortunately, there is no way to customize the name,